### DISCUSSION TITLE
Please add day of the week on commit date

### DISCUSSION THREAD

**Discussion by warsngl** [community/community#49989] [Actions]
### Select Topic Area

Question

### Body

now - Commits on Mar 3, 2023

better - Commits on Mar 3, 2023 Friday

**Comment by Anmol-Baranwal** [comment]
Actually, it would be even better to format it as "Commits on Friday, Mar 3, 2023" to make it more readable and easier to understand at a glance. So the revised format would be:

"Commits on Friday, Mar 3, 2023" on GitHub.

**Reply by Anmol-Baranwal** [reply]
@warsngl 
This issue should be closed, and you should do this in product feedback.

If you have no further doubts or if your doubts have been resolved, would you mind marking my answer as accepted?
